src/ctl/src/lib.rs

```diff
@@ -847,7 +872,15 @@ async fn start_impl(opts: CliOpts, context: &CtlContext) -> Result<()> {
         Commands::Meta(MetaCommands::GraphCheck { endpoint }) => {
             cmd_impl::meta::graph_check(endpoint).await?
         }
-        Commands::AwaitTree => cmd_impl::await_tree::dump(context).await?,
+        Commands::AwaitTree(AwaitTreeCommands::Dump {
+            actor_traces_format,
+        }) => cmd_impl::await_tree::dump(context, actor_traces_format).await?,
+        Commands::AwaitTree(AwaitTreeCommands::Analyze { path }) => {
+            cmd_impl::await_tree::bottleneck_detect(context, path).await?
+        }
+        Commands::AwaitTree(AwaitTreeCommands::Transcribe { path }) => {
+            cmd_impl::await_tree::transcribe(path)?
```

Comment:
still slightly prefer to put these logic in a separate repo. (It's ok to let risectl include them, but they can be developed separately and iterate faster. Also no need to compile risectl which means compiling whole risingwave to develop)

Reply:
Done. https://github.com/risingwavelabs/rw-diagnose-tools